save-plugins/fix-plugin/src/commonMain/kotlin/com/saveourtool/save/plugins/fix/FixPlugin.kt

```diff
@@ -230,10 +230,7 @@ class FixPlugin(
     ): List<TestResult> = testCopyToExpectedFilesMap.map { (testCopy, expected) ->
         val fixedLines = fs.readLines(testCopy)
         val expectedLines = fs.readLines(expected)
-
-        // FixMe: https://github.com/saveourtool/save-cli/issues/473
-        val test = testToExpectedFilesMap.first { (test, _) -> test.name == testCopy.name }.first
-
+        val test = testToExpectedFilesMap.first { (test, _) -> test.createRelativePathToTheRoot(testConfig.directory) in testCopy.toString() }.first
```

Comment:
what with comparing by paths? and not strings?

Reply:
Yes. Fixed it

Reply:
Just compare the paths, starting from the directory, which is the **parent** for the root `save.toml`, think this should work